### DISCUSSION TITLE
Converting a string to JSON without "/"

### DISCUSSION THREAD

**Discussion by Kolhun** [nlohmann/json#4317] [Q&A] [answered]
Hello everyone I'm trying to write the string "vars": {"category":"44"} to json, but for some unknown reason it writes iterators / to the string, the code below attempts to fix this problem (I didn't see any obvious solutions to the problem in the JSON documentation)
```cpp
json _dataPost;
_dataPost["vars"] = R"({"category":")" + param1 + R"("})";
```
conclusion
![image](https://github.com/nlohmann/json/assets/36815965/85ef0728-fcb5-4f48-af85-11a77ded4499)

Tell me how to solve this problem...

**Comment by Kolhun** [comment]
I tried another way, for some reason I wrote down another \n characters

```cpp
json demo;
demo["vars"] =
{
    {"category", param1.c_str()}
};
std::string ss = demo["vars"].dump(0);
paramStr = ss.c_str();

json _dataPost;
_dataPost["vars"] = paramStr.c_str();
```

![image](https://github.com/nlohmann/json/assets/36815965/03543b72-9a2a-4706-adc3-c9cbe70b621b)

**Comment by gregmarr** [comment, marked as answer]
```
json _dataPost;
_dataPost["vars"] = { "category", param1 };
```
https://www.godbolt.org/z/1f76rWsej

**Comment by Kolhun** [comment]
> ```
> json _dataPost;
> _dataPost["vars"] = { "category", param1 };
> ```
> 
> https://www.godbolt.org/z/1f76rWsej
![image](https://github.com/nlohmann/json/assets/36815965/068b2566-08df-4756-9a2e-911eda4976c1)

Need "vars": {"category":"124325"},
